src/migrate/sources/fs-migrations.js

```diff
@@ -3,7 +3,7 @@ import path from 'path';
 import Promise from 'bluebird';
 import { sortBy, filter } from 'lodash';
 
-const readDirAsync = Promise.promisify(fs.readdir, { context: fs });
+const readDirAsync = (path) => Promise.promisify(fs.readdir, { context: fs })(path);
 
```

Comment:
Import above doesn't break already?

Reply:
Supposedly not, that was my first thought as well. Seems its set to empty object instead, presumably by webpack.